README.md

```diff
@@ -1,10 +1,31 @@
 # Pulplift
 
-This is based on and makes use of the Vagrant based configuration of the [Forklift](https://github.com/theforeman/forklift) project. Please see the Forklift documentation for all configuration options available when creating custom or adding new boxes.
+This is based on and makes use of the Vagrant based configuration of the
+[Forklift](https://github.com/theforeman/forklift) project. Please see the Forklift documentation
+for all configuration options available when creating custom boxes.
+
+## Quickstart
+
+To setup Pulplift, ensure that all submodules are updated and in place.
+
+```
+git submodule update --init
+```
+
+You will also need to [clone the source
+code](https://docs.pulpproject.org/en/3.0/nightly/contributing/dev-setup.html#get-the-source).
```

Comment:
This link is currently broken, but will work after the docs builder for pulpcore runs tonight.

Reply:
i suggest to add to th get the source section a note that will hight that some of the plugins cannot be installed together if the are pinned on different pulpcore-pluging version

Reply:
That's what I had in mind with this warning.
https://github.com/pulp/pulpcore/pull/54/files#diff-9626d9b912838bba64a63d99f0854336R33

I could specifically mention that this is more likely to occur with plugins requiring specific versions of pulpcore-plugin. Would that be what you had in mind?

Reply:
this is exactly what i had in mind, i just have not reviewed the other PR yet